src/AirtableRecord.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Yoanbernabeu\AirtableClientBundle;
+
+use DateTime;
+use DateTimeInterface;
+
+class AirtableRecord
+{
+    protected $fields;
+    protected string $id;
+    protected DateTimeInterface $createdTime;
+
+    public function __construct(string $id, $fields, DateTimeInterface $createdTime)
+    {
+        $this->fields = $fields;
+        $this->id = $id;
+        $this->createdTime = $createdTime;
+    }
+
+    public static function fromRecord(array $record): self
```

Comment:
rename to `create`? or `createFromRecord`?

Reply:
createFromRecord ! Done